src/Shared/ThrowHelpers/ArgumentThrowHelper.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.AspNetCore.Shared;
+
+#nullable enable
+
+internal static partial class ArgumentThrowHelper
+{
+#if !NET7_0_OR_GREATER || NETSTANDARD ||NETFRAMEWORK
+    private const string EmptyString = "";
```

Comment:
o.O `string.Empty`

Reply:
Ah yeah, string.Empty is available in the versions targeted here so it should be fine.

Reply:
Took me a while to figure out why this was needed. `string.Empty` isn't a `const` so you can't use it in equality checks. We also cannot use `string.IsNullOrEmpty` because it isn't annotated for nullability in netfx and netstandard.

Reply:
> can't use it in equality checks

o.O
You mean can't use it with `or`? Why not just use `||` then?

Reply:
Merp -- thought that was what I tried in https://github.com/dotnet/aspnetcore/pull/48243/commits/b09309adc4e497b14f7a17608f557c0735084488 but looks like I was wrong.